source/tool/chipsec/hal/acpi_tables.py

```diff
@@ -584,3 +584,27 @@ def __str__( self ):
 ACPI Table Entries:
 %s
 """ %( ''.join( ['0x%016X\n' % addr for addr in self.Entries]))
+
+########################################################################################################
+#
+# FACP Table

```

Comment:
Sure, was just confused I guess since I thought the source code only refers to table signature instead of table name.